mlrun/platforms/other.py

```diff
@@ -165,3 +166,75 @@ def _mount_hostpath(task):
         )
 
     return _mount_hostpath
+
+
+def mount_s3(
+    aws_access_key="",
+    aws_secret_key="",
+    secret_name=None,
+    endpoint_url=None,
+    aws_region=None,
```

Comment:
Why is this needed? I don't recall we're using it anywhere, and in general S3 access is not region-governed (each bucket is in a region, but you don't specify a region when accessing it)

Reply:
i copied it from the s3 modifier in KFP, i assume it has usage